config/application.rb

```diff
@@ -14,7 +14,7 @@ class Application < Rails::Application
     config.load_defaults 7.0
 
     config.autoload_paths += %W(#{config.root}/lib)
-    
+    config.eager_load_paths += %W(#{config.root}/lib)
```

Comment:
why was this needed?

Reply:
not sure? `rails zeitwerk:check` suggested it done. Is it redundant with `autoload_paths`?

Reply:
eager_load I believe forces all those files loaded on boot, autoload is a rails magic thing that does lazy class loading (it works by catching constant not defined errors and then trying to look for files matching the right naming conventing in all the autoload paths).  In production, you'd want to just load everything upfront, in development autoload keeps boot fast and only loads things as you need since you don't care about performance